warplib/memory.cairo

```diff
@@ -119,6 +126,143 @@ func wm_dyn_array_length{warp_memory : DictAccess*}(arrayLoc : felt) -> (len : U
     return (Uint256(low, high))
 end
 
+func wm_bytes_new{range_check_ptr, warp_memory : DictAccess*}(len : Uint256) -> (loc : felt):
```

Comment:
As glorious as this all is it should probably be commented/moved to another file to be clear it's for future use and not for now